esx_service/utils/vmdk_utils.py

```diff
@@ -77,6 +77,7 @@ def init_datastoreCache(force=False):
     with lockManager.get_lock("init_datastoreCache"):
         global datastores
         logging.debug("init_datastoreCache:  %s", datastores)
+
```

Comment:
nit: please remove this file from this PR that avoid confusion in future for just new line change